src/include/rollback_to_stable.h

```diff
@@ -81,6 +82,16 @@
             WT_STAT_CONN_DATA_INCR(session, stat##_dryrun); \
     } while (0)
 
+/*
+ * WT_RTS_WORK_UNIT --
+ *	A definition of maintenance that a RTS tree needs done.
```

Comment:
I don't think we can use "RTS tree" as RTS is not a typ of tree. I see that this definition exists for `LSM trees` and `tiered trees`, which both make sense as they mention actual types.
I don't have a suggestion yet but we need to update the comment imo.